jupyterhub_singleuser_profiles/api/api.py

```diff
@@ -85,6 +85,10 @@ def get_images(*args, **kwargs):
     images = _PROFILES.images.get()
     return images
 
+@authenticated
+def get_image_info(image_name, *args, **kwargs):
+    return _PROFILES.get_image_info(image_name)
```

Comment:
Should this return some kind of error when the image does not exist? Like not return 202, but rather 404?